package.json

```
@@ -24,8 +24,8 @@
     "gql": "graphql-codegen --config codegen.yml",
     "gql:w": "yarn gql --watch",
     "postinstall": "husky install",
-    "onesky:upload": "babel-node ./onesky/upload.js",
-    "onesky:download": "babel-node ./onesky/download.js",
+    "crowdin:sort-translations": "babel-node ./crowdin/sort-translations.js",
```

This isn't necessarily part of your PR, but we could remove the `babel-node` dependency and run this script directly.

```suggestion
    "crowdin:sort-translations": "node ./crowdin/sort-translations.js",
```